amy/api/tests/test_api_structure.py

```diff
@@ -1,6 +1,7 @@
 from django.urls import reverse
 from rest_framework.test import APITestCase
 
+from amy.workshops.tests.base import consent_to_all_required_consents
```

Comment:
Please fix to `workshops.tests.base` (without `amy.`).

Reply:
Oh hmm.. thought I'd fixed that in my editor. Good catch.